examples/compiled/rect_heatmap.vg.json

```diff
@@ -179,11 +179,6 @@
         },
         "axisY": {
             "minExtent": 30
-        },
```

Comment:
The default make sense for everything but the heatmap :-P I will change the example.